native/src/utils/PushNotificationsManager.ts

```diff
@@ -8,9 +8,17 @@ import { NavigationPropType, RoutesType } from '../constants/NavigationTypes'
 import buildConfig from '../constants/buildConfig'
 import navigateToDeepLink from '../navigation/navigateToDeepLink'
 import urlFromRouteInformation from '../navigation/url'
-import appSettings from './AppSettings'
 import { log, reportError } from './sentry'
 
+type Message = FirebaseMessagingTypes.RemoteMessage & {
+  data: {
+    city_code: string
+    language_code: string
+    group: string
```

Comment:
group is `news` or `tünews`? Should we maybe make this type more strict?

Reply:
`group` is something like `testumgbung-de-news` and not used. Will therefore remove this to avoid confusion.